README.md

```diff
@@ -400,6 +400,7 @@ plugins to process and writes them to a queue.
 | `SYNC_DATA` | When the database provider is set to `local` this will be where the data is stored relative to the `/srv/data` working directory. Default is `./db` |
 | `SYNC_DATABASE_DOCUMENT_PATH` |  When the database provider is set to `firestore` this value is the path to the document in Cloud Firestore. Must be in the form of `<collection>/<document>`. Default is `sync-server/wporg`. |
 | `SYNC_DATABASE_PROVIDER` | Tells the Sync Server which database provider to use; either the local file system or Google Cloud Firestore. Must be one of: `local`, `firestore`. Default is `local`. |
+| `SYNC_MESSAGE_PROVIDER` | Tells the Sync Server which message/queue provider to use; either the AWS SQS or Google Cloud Firestore. Must be one of: `aws`, `firestore`. Default is `aws`. |
```

Comment:
We have established patterns for these config values in other environment variables so this should be one of `local`, `sqs`, or `firestore` and the default should be `local` once we integrate MongoDB. However, if we were to use `aws` then `firestore` would become `gcp`.

Reply:
thats fair.... will fix in the PR